### ISSUE TITLE
Giftschrank-Flag

### ISSUE THREAD

**Issue by MaZderMind** [voc/voctoweb#168]
There should be a a Flag on the Event-Model which completely hides the Event from all UIs.

**Comment by derpeter** [comment]
this would also be a very nice feature for releasing. It happens often that we are waiting for an "ok" before things can go public which at the moment means staling the release process.

**Comment by manno** [comment]
I don't get it, we need to delete the files anyways? 
Additionally sometimes the video has to be deleted, but the audio may  still be ok, because it does not contain copyrighted image.s

**Comment by QbDesu** [comment]
I agree with @manno on this. If something needs to be unpublished, even if temporary the files need to be removed from the CDN as well anyway. At that point we might as well just delete the event or all of it's recordings and republish it when the situation at hand was resolved. What do we lose, some counted views and the original publishing date maybe, nothing too bad.
